provision/acc_provision/acc_provision.py

```diff
@@ -160,6 +160,7 @@ def config_default():
             "kube_default_provide_kube_api": False,
             "disable_node_subnet_creation": False,
             "preexisting_kube_bd": None,
+            "device_delete_timeout": None,
```

Comment:
Changes look good but I would like to suggest a minor change - can we rename the config to opflex_device_delete_timeout

Reply:
addressed